src/main/cpp/t10_unique.cpp

```diff
@@ -21,9 +21,28 @@
 
 #include "t10_unique.h"
 #include <iostream>
-
-using namespace std;
+#include <vector>
+//using namespace std;
 
 int t10_unique() {
+    int n, c = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'c' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4718004)